OpenTabletDriver.Configurations/Configurations/Huion/Note X10.json

```
@@ -0,0 +1,32 @@
+{
+  "Name": "Huion Note X10",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 186.9,
+      "Height": 140.9,
```

based on the `MaxX`/`MaxY` coordinates given
```suggestion
      "Width": 187,
      "Height": 141,
```